WINDOWS.md

```diff
@@ -354,9 +354,25 @@ When the installation is finished, launch VS Code.
 
 ### Connecting VS Code to Ubuntu
 
-You should see a frame at the bottom-right corner of VS Code saying it detects WSL. Accept the installation of the WSL extension for VS Code.
+To make VS Code interact properly with Ubuntu, let's install the [Remote - WSL](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl) VS Code extension.
 
-:warning: This extension is mandatory for you to be able to work with VS Code!
+Copy-paste the following commands in your terminal:
+
+```bash
+code --install-extension ms-vscode-remote.remote-wsl
+```
+
+Then open VS Code from your terminal:
+
+```bash
+code .
+```
+
+:heavy_check_mark: If you see `WSL: Ubuntu` in a green box in the bottom left corner of the VS Code window, you're good to go :+1:
+
+![WSL Ubuntu Remote](https://github.com/lewagon/setup/blob/master/images/windows_remote_wsl.png)
+
+:x: Otherwise, please **contact a teacher**
```

Comment:
This is one of the main change. It appears that the WSL Remote VS Code extension popup is not showing anymore. 

So I added clear instructions on how to install this necessary extension.